python/setup.cfg

```diff
@@ -44,6 +44,7 @@ packages = find:
 python_requires = >=3.8
 install_requires =
     mmh3
+    pytz
```

Comment:
Is this needed? We have time utils already.

Reply:
Updated the PR and removed it.